### ISSUE TITLE
Why set the NUM_OF_CLASSESS = 151

### ISSUE THREAD

**Issue by MZ59** [shekkizh/FCN.tensorflow#69]
I have try to set the NUM_OF_CLASSESS = another value，the result is loss = nan.
Would you please tell me the reason why the NUM_OF_CLASSESS must equal to 151?
Thank you!

**Comment by xylcbd** [comment]
because the categories of the dataset is 150, plus 'Unknown', final the classes is 151.

**Comment by kong869** [comment]
Hello，the value of NUM_OF_CLASSESS make me confused. If my categories of dataset is 0:road 1:building 2:background, the value of NUM_OF_CLASSESS is 3. Right? in my situation, the 'background' contains the 'Unknown', am i right? Thanks in advance. (BTW: it is running well as i said)
